third_party/libpg_query/grammar/keywords/reserved_keywords.list

```diff
@@ -8,6 +8,7 @@ AS
 ASC_P
 ASYMMETRIC
 BOTH
+CASCADE
```

Comment:
We would prefer not to add `CASCADE` or `RESTRICT` these to the `reserved_keywords` list if possible. We want to be very careful with adding new reserved keywords since new reserved keywords can break existing SQL programs.

Reply:
Reverted the change made that allowed CASCADE on ALTER TYPE 👍